### DISCUSSION TITLE
[BUG] Fastfetch fails to run on Windows 7 despite what the README says

### DISCUSSION THREAD

**Discussion by ghost** [fastfetch-cli/fastfetch#1279] [Q&A] [answered]
Be sure to read the [FAQ](https://github.com/fastfetch-cli/fastfetch?tab=readme-ov-file#faq) before submitting a new issue.

# General description of bug:

* What happened: fastfetch cannot start because api-ms-win-crt-convert-l1-1-0.dll is missing
* What should happen: fastfetch starts normally
* Fastfetch version used: 2.25.0
* Did it work in an older version: no
* Where did you get the binary: GitHub release
* Does this issue still occur in [the latest dev build](https://github.com/fastfetch-cli/fastfetch/actions/)? yes

# Often helpful information:

Screenshot:
![image](https://github.com/user-attachments/assets/c6377701-d9e4-4255-93e8-f6c4fd9d9a03)

Output of `fastfetch -c ci.jsonc --format json`:

cannot provide, fastfetch doesn't start

Output of `fastfetch --list-features`:
cannot provide, fastfetch doesn't start

**Comment by CarterLi** [comment]
You should install UCRT runtime library. See https://github.com/fastfetch-cli/fastfetch/wiki/Dependencies#windows

**Reply by ghost** [reply]
unfortunately this gives an error:
"Installer encountered an error: 0x80070003 The system cannot find the path specified."

**Reply by ghost** [reply, marked as answer]
fixed by installing vc redist from https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170
